src/main/java/org/jboss/logmanager/ExtHandler.java

```diff
@@ -307,16 +266,11 @@ protected static void checkAccess() throws SecurityException {
      * @param handler the handler to check access on.
      *
      * @throws SecurityException if a security manager exists and if the caller does not have {@code
-     *                           LoggingPermission(control)} or the handler is {@link #protect(Object) protected}.
+     *                           LoggingPermission(control)}
      */
+    @Deprecated
     protected static void checkAccess(final ExtHandler handler) throws SecurityException {
-        final SecurityManager sm = System.getSecurityManager();
-        if (sm != null) {
-            sm.checkPermission(CONTROL_PERMISSION);
-        }
-        if (handler.protectKey != null && handler.granted.get() == null) {
-            throw accessDenied();
-        }
+        checkAccess();
```

Comment:
Since we removed the other methods we could probably just remove this at some point too.